tools/Utils/c_rehash.py

```diff
@@ -4,36 +4,55 @@
 import argparse
 import subprocess
 
-def process_file (dirname, file) :
+
+def process_file(dirname, file):
+## process_file () function
+#   - creates copy of certificate named with hash of inner certificate data
+#  @file - file for processing 
+
     # Old full file ocation
-    full_path = dirname + "\\" + file
+    full_path = os.path.join(dirname, file)
```

Comment:
@Kozoriz  file is python reserved variable, probably it would be good to avoid this name